ui/apps/platform/src/Containers/NetworkGraph/TopologyComponent.tsx

```diff
@@ -173,11 +174,11 @@ const TopologyComponent = ({
         controller.fromModel(model);
         if (selectedNode) {
             panNodeIntoView(selectedNode);
-        } else if (history.location.pathname !== networkBasePath && !selectedNode) {
```

Comment:
Aha, both `history.location.pathname` and `location.pathname` in React Router 5 are worth mention in the next team meeting after recharge. Pro and con of providing synonyms, and then removing them later.